exporter/otlphttpexporter/otlp.go

```diff
@@ -131,6 +143,15 @@ func (e *baseExporter) pushMetrics(ctx context.Context, md pmetric.Metrics) erro
 	return e.export(ctx, e.metricsURL, request, e.metricsPartialSuccessHandler)
 }
 
+func (e *baseExporter) pushMetricsWithStatus(ctx context.Context, md pmetric.Metrics) error {
+	if err := e.pushMetrics(ctx, md); err != nil {
+		componentstatus.ReportStatus(e.host, componentstatus.NewRecoverableErrorEvent(err))
+		return err
+	}
+	componentstatus.ReportStatus(e.host, componentstatus.NewEvent(componentstatus.StatusOK))
```

Comment:
this might create a lot of events reporting OK, is that really what you want? Is it there to recover from the error status?